### PR TITLE
Add formencode

### PR DESCRIPTION
**Pull request by windelbouwmanbosch**

### CODE REVIEW

File: recipes/formencode/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - six
```
Comment (synapticarbors) [inline recipes/formencode/meta.yaml]: It doesn't look like you need `six` as a `build` or `run` dependency. I don't see it mentioned in any of the source in the tarball. I think the version on github (not released to pypi) does make use of it.

File: recipes/formencode/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    - python
+    - six
+
+test:
+  imports:
+    - formencode
+
+about:
+  home: http://formencode.org
+  license: MIT
```
Comment (synapticarbors) [inline recipes/formencode/meta.yaml]: Can you manually package the license:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

and include the following issue link as a comment in the reipe:
https://github.com/formencode/formencode/issues/91

File: recipes/formencode/meta.yaml (hunk 1 above)
Comment (windelbouwmanbosch) [reply recipes/formencode/meta.yaml]: Okay, good point, I'll remove it.

**Comment by synapticarbors** [comment]
@windelbouwmanbosch -- Thanks for your contribution to conda-forge. A feedstock for this recipe should be generated soon and you'll be receiving several automated messages about it. When the feedstock has been rendered, please take a look at the README in that repo for information about updating and maintaining the recipe. One of the key points is all updates should be done by forking that repo and following a standard pull request workflow. If you haven't looked at it already the docs are quite helpful:

https://conda-forge.org/docs/

If you run into any difficulties please let us know.

**Pull request merged**